img/dist/icon-remove.svg

```diff
@@ -0,0 +1 @@
+<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 45 45"><title>close</title><path vector-effect="non-scaling-stroke" d="M40 40L22.5 22.5 40 5"/><path vector-effect="non-scaling-stroke" class="cls-1" d="M5 5l17.5 17.5L5 40"/></svg>
```

Comment:
not worth changing but feel like expanded strokes into fills can be easier to use to avoid rendering/style issues as I see the mitre rules in the css etc